web-registry/src/pages/CreditBatchBasics.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { CreditBatchFormTemplate } from '../components/templates';
+import { BatchBasicsForm } from '../components/organisms';
+
+const CreditBatchBasics: React.FC = () => {
+  return (
+    <CreditBatchFormTemplate title="Create Credit Batch" activeStep={0}>
+      <BatchBasicsForm submit={() => Promise.resolve()} />
```

Comment:
we should probably do something, to save the current form state in the application state and ideally in local storage too

Reply:
nevermind, as discussed in https://www.notion.so/regennetwork/Interfaces-Architecture-2022-05-11-56d7950ca0fa4224b313ea64d97e594c, that should tackled as part of https://github.com/regen-network/regen-registry/issues/899